resources/styles/tutor.less

```diff
@@ -39,7 +39,6 @@
 @import './components/course-listings';
 @import './components/tutor-dialog';
 @import './components/task/index';
-@import './components/spy-mode';
 @import './components/task-step/index';
```

Comment:
I've removed the built in styles here, but the line below (you'll have to expand diff, it's line 46) that requires them from openstax-react-components seems incorrect.  There's not an assets/main.css file unless they're built using webpack.  Shouldn't we require the styles like how concept coach does by: `@import '~openstax-react-components/resources/styles/main';`